app/javascript/controllers/master_detail_controller.js

```diff
@@ -0,0 +1,8 @@
+import { Controller } from 'stimulus'
+
+export default class extends Controller {
+  select () {
+    debugger
+    CodeFundTheme.toggleSidebar()
```

Comment:
Debugger needs to be removed and we probably should actually figure out why this is happening vs this hack.